frontend/src/metabase/query_builder/components/view/sidebars/ChartTypeSidebar.styled.tsx

```diff
@@ -2,7 +2,7 @@ import styled from "@emotion/styled";
 import { color, lighten, tint, isDark } from "metabase/lib/colors";
 
 export interface OptionRootProps {
-  isSensible?: boolean;
+  isSelected?: boolean;
```

Comment:
nitpick, but in my view `isSensible` was fine here because `isSelected` more suitable for an actual selected viz type